synthetics_accountApp.tf

```diff
@@ -1,24 +0,0 @@
-resource "datadog_synthetics_test" "accountsApp" {
-  type = "api"
-  request_definition {
-    method = "GET"
-    url    = "https://accounts.jenkins.io"
-  }
-  assertion {
-    type     = "statusCode"
-    operator = "is"
-    target   = "200"
-  }
-  locations = ["aws:eu-central-1"]
-  options_list {
-    tick_every = 900
-  }
-  name    = "account.jenkins.io"
-  message = "Notify @pagerduty"
-  tags = [
-    "jenkins.io",
-    "production"
-  ]
-
-  status = "live"
-}
```

Comment:
I'm wondering if we should keep it or not.

If the new monitors fails, then it means that either the acount application is down (Kubernetes down/DNS/ no LB/App failing/OOM/etc.) and / OR LDAP is down.

I wonder if we shouldn't keep both to ensure that we can quickly, just by staring at the monitors, where is the "area of failure"

WDYT?

Reply:
(sorry for the additional work, I did not thought about it on first review)